charts/vector/README.md

```diff
@@ -1,6 +1,6 @@
 # Vector
 
-![Version: 0.42.1](https://img.shields.io/badge/Version-0.42.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.46.1-distroless-libc](https://img.shields.io/badge/AppVersion-0.46.1--distroless--libc-informational?style=flat-square)
+![Version: 0.43.0](https://img.shields.io/badge/Version-0.43.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.47.0-distroless-libc](https://img.shields.io/badge/AppVersion-0.47.0--distroless--libc-informational?style=flat-square)
```

Comment:
I think you actually don't want to do this yet, but instead do it when creating the release PR.